src/sentry/templates/sentry/emails/_group.html

```diff
@@ -3,46 +3,47 @@
 {% with type=group.get_event_type metadata=group.get_event_metadata transaction=group.culprit %}
 <div class="issue {% if group.is_resolved %}resolved{% endif %}">
   {% if type == "error" %}
-    <div class="event-type error">
-      <h3>
-        {% if metadata.type %}
-          {% include "sentry/emails/_group_link.html" with text=metadata.type|truncatechars:40 %}
-          {% if transaction %}
-            <span class="event-subtitle">{{ transaction }}</span>
-          {% endif %}
-          <br />
-          {% if metadata.value %}
-            <small>{{ metadata.value|truncatechars:100|soft_break:40 }}</small>
-          {% endif %}
-        {% else %}
-          {% include "sentry/emails/_group_link.html" with text=metadata.value|truncatechars:40 %}
-          <br />
-          {% if transaction %}
-            <span class="event-subtitle">{{ transaction }}</span>
-          {% endif %}
-        {% endif %}
-      </h3>
-    </div>
+  <div class="event-type error">
+    <h3>
+      {% if metadata.type %}
+      <a href="{{group.get_absolute_url}}">{{metadata.type|truncatechars:40}}</a>
+      {% include "sentry/emails/_group_link.html" with text=metadata.type|truncatechars:40 %}
+      {% if transaction %}
+      <span class="event-subtitle">{{ transaction }}</span>
+      {% endif %}
+      <br />
+      {% if metadata.value %}
+      <small>{{ metadata.value|truncatechars:100|soft_break:40 }}</small>
+      {% endif %}
+      {% else %}
+      <a href="{{group.get_absolute_url}}">{{metadata.value|truncatechars:40}}</a>
+      <br />
+      {% if transaction %}
+      <span class="event-subtitle">{{ transaction }}</span>
+      {% endif %}
```

Comment:
Indentation here makes the template harder to read than how it was before.

Reply:
Ugh prettier does this, will fix

Reply:
We may want to add a `.prettierignore` file with `*.html`